src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -198,7 +198,10 @@ internal class CommandLineParameterParser
 
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-windows`
Telemetry for shell startup.Error occurred in Describe block
```powershell
You cannot call a method on a null-valued expression.
At D:\a\1\s\test\powershell\engine\Basic\Telemetry.Tests.ps1:12 char:9
```